languages/tree-sitter-stack-graphs-java/README.md

```diff
@@ -0,0 +1,27 @@
+# tree-sitter-stack-graphs definition for Java
+
+This project defines tree-sitter-stack-graphs rules for Java using the [tree-sitter-java](https://www.npmjs.com/package/tree-sitter-java) grammar.
+
+## Development
+
+The project is organized as follows:
+
+- The stack graph rules are defined in `src/stack-graphs.tsg`.
+- Tests are put into the `tests` directory.
+
+Make sure all development dependencies are installed by running:
+    ./bootstrap
+
+Run all tests in the project by executing the following:
+
+    cargo test
+
+Parse and test a single file by executing the following commands:
+
+    <!-- TODO: allow this option via cargo -->
```

Comment:
This is possible now (and reviewing this reminds me that the README for TypeScript needs some serious fixing). Parsing a file:

    cargo run -- parse OPTIONS FILENAME

Running a test:

    cargo run -- test OPTIONS FILENAMES

It is also possible to use `./target/tree-sitter-stack-graphs-java` (after `cargo build`) or `tree-sitter-stack-graphs-java` (after `cargo install`) instead of `cargo run --`. For the README, I would describe the `cargo run` options for local development, and `tree-sitter-stack-graphs-java` for installed use.

Reply:
I updated the README with these suggestions but modified the command a little to make it run locally for me. If I missed something to make those commands work as supplied, I'm happy to try and update accordingly.

I still need to update the docs for installed use.